src/lib/pubkey/kyber/kyber_common/kyber_symmetric_primitives.h

```diff
@@ -41,7 +33,7 @@ class Kyber_Symmetric_Primitives {
       virtual std::unique_ptr<HashFunction> H() const = 0;
       virtual std::unique_ptr<HashFunction> KDF() const = 0;
 
-      virtual std::unique_ptr<Kyber_XOF> XOF(std::span<const uint8_t> seed) const = 0;
+      virtual Botan::XOF& XOF(std::span<const uint8_t> seed, std::tuple<uint8_t, uint8_t> matrix_position) const = 0;
```

Comment:
No need to namespace prefix within `Botan`

Reply:
Some compilers are confused about the double meaning of the identifier:

```
  botan_all.cpp:8275:20: error: declaration of ‘virtual Botan::XOF& Botan::Kyber_Symmetric_Primitives::XOF(std::span<const unsigned char>, std::tuple<unsigned char, unsigned char>) const’ changes meaning of ‘XOF’ [-fpermissive]
   8275 |       virtual XOF& XOF(std::span<const uint8_t> seed, std::tuple<uint8_t, uint8_t> matrix_position) const = 0;
        |                    ^~~
```

Reply:
WTF???

Anyway fine to leave like that if not prefixing is causing problems otherwise.